### ISSUE TITLE
After closing Anki in Anki 24.04RC3, residual processes appear, which prevents Anki from starting again.

### ISSUE THREAD

**Issue by WeltKarl** [C9HDN/anki-button-colours#9]
At first, I thought it was a bug in the new version of Anki itself, so I immediately reported it to the author of Anki. He asked me to disable the add-ons to see if I could reproduce the problem, and it turned out that "Button Colours Good Again" was causing the issue.
![image](https://github.com/teaqu/anki-button-colours/assets/145196400/73c9dab0-4e47-409d-9e79-66e50e637615)
![image](https://github.com/teaqu/anki-button-colours/assets/145196400/31c930f8-0f40-4a73-b154-2a98711b650a)

**Comment by WeltKarl** [comment]
Hello. I just tested it again on the 24.04 official version and the latest 24.04.1rc1 test version, and the issue still persists.

**Comment by teaqu** [comment]
Hello,

I can't seem to reproduce this issue with a fresh install of 24.04.1 🤔

**Comment by WeltKarl** [comment]
Hello,

I've tested it several more times upon seeing your reply. The issue occurred again, but this time it was with a different add-on, which is very strange because it didn't cause an error last time. Everything worked fine after I disabled this add-on. Then I clicked "Check Database," and the problem appeared again...

This issue is peculiar, and it seems that the root cause is still with the new version of Anki, not your add-ons. I think I need to report this strange problem to the author of Anki again.

Thank you very much for creating this add-on, it is truly useful. I apologize for disturbing you due to a bug that is not related to your work, and have a nice day！
